cmake/modules/AddSwift.cmake

```diff
@@ -700,6 +701,13 @@ function(_add_swift_library_single target name)
       INSTALL_IN_COMPONENT "${SWIFTLIB_INSTALL_IN_COMPONENT}")
   add_swift_source_group("${SWIFTLIB_SINGLE_EXTERNAL_SOURCES}")
 
+  # If there were any swift sources, then a .swiftmodule may have been created.
+  # If that is the case, then add a target which is an alias of the module files.
+  if(NOT "${module_target}" STREQUAL "" AND NOT "${swift_module_dependency_target}" STREQUAL "")
```

Comment:
If you made module_target a cmake_parse_arguments target, then this would be cleaner. Specifically, you could do:

if (NOT "${SWIFTLIB_MODULE_TARGET}" AND NOT "${swift_module_dependency_target}")

endif()

Then it would be clear that module_target is an argument due to the SWIFTLIB_* prefix.